README.md

```diff
@@ -100,7 +100,7 @@ fastatic({ src: 'my-static-site-source/', dest: 'my-static-site-dest/' });
 
 ## Contributing
 
-See [CONTRIBUTING.md](CONTRIBUTING.md) for [guidelines](CONTRIBUTING.md#guidelines) and [development scripts](CONTRIBUTING.md#scripts).
+See [CONTRIBUTING.md](https://github.com/voorhoede/fastatic/blob/master/CONTRIBUTING.md) for [guidelines](https://github.com/voorhoede/fastatic/blob/master/CONTRIBUTING.md#guidelines) and [development scripts](https://github.com/voorhoede/fastatic/blob/master/CONTRIBUTING.md#scripts).
```

Comment:
This won't work on other branches and is not really friendly to forks as they will always point to our master. I think it's cleaner to use Marked programmatically to solve this. You can override the [`link` renderer](https://github.com/chjj/marked#inline-level-renderer-methods), test if it doesn't start with `https?://` and if not prefix it with `https://GH_USER/GH_REPO/`. (You can check ocelot for how we're overriding default renderers).

As we are getting more scripts just around the examples, maybe move them all to `scripts/` (e.g. `scripts/build-index`, `scripts/rebase-react` etc)